Monika After Story/game/zz_backgrounds.rpy

```diff
@@ -3216,7 +3216,17 @@ label monika_change_background_loop:
         skip_transition = mas_background.EXP_SKIP_TRANSITION in sel_background.ex_props
         skip_outro = mas_background.EXP_SKIP_OUTRO in sel_background.ex_props
 
+    # UI shields + buttons
+    # NOTE: buttons are in here since there is no consistency if placed in 
+    # the bg change label.
+    $ mas_RaiseShield_core()
+    $ HKBHideButtons()
+
     call mas_background_change(sel_background, skip_leadin=skip_leadin, skip_outro=skip_outro, set_persistent=True)
```

Comment:
```suggestion
    call mas_background_change(sel_background, skip_leadin=skip_leadin, skip_transition=skip_transition, skip_outro=skip_outro, set_persistent=True)
```
We set `skip_transition` a few lines before but we never pass it in `mas_background_change`. We should probably do so just in case a bg needs it in the future.